python_modules/dagster-framework/pyspark/dagster_framework/pyspark/__init__.py

```diff
@@ -0,0 +1,89 @@
+# getting false positives in vscode
+from pyspark.sql import SparkSession  # pylint: disable=no-name-in-module,import-error
+from pyspark.rdd import RDD  # pylint: disable=no-name-in-module,import-error
+
+from dagster import (
+    Bool,
+    Dict,
+    Field,
+    Selector,
+    Path,
+    String,
+    as_dagster_type,
+    check,
+    input_selector_schema,
+    output_selector_schema,
+    resource,
+)
+from dagster.core.types.runtime import define_any_type
```

Comment:
doesnt look like this ends up getting used? remove?

Reply:
yup. need to start linting these